.changeset/fifty-fireants-judge.md

```diff
@@ -0,0 +1,20 @@
+---
+'@commercetools-frontend/application-shell': minor
+---
+
+Add a new `<SuspendedRoute>` component.
+
+This component should be used to render a lazy loaded component.
```

Comment:
```suggestion
This component is exactly like the `<Route>` component of `react-router` but it wraps the children with `React.Suspense`, allowing the children to be dynamically loaded with `React.lazy`. This can be used for code splitting components at the route level.
```